cmd/set.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redhat-developer/ocdev/pkg/component"
+	"github.com/spf13/cobra"
+)
+
+var componentSetCmd = &cobra.Command{
+	Use:   "set",
```

Comment:
This command should be called `component` and should be coupled together with `set`

```
# getting current component
ocdev component

# switching current component 
ocdev component <name>
```

if you look `oc project` in works in the same way

Reply:
There will be no `set` command. 
`ocdev component <name>` will act as set. If there is no argument, it will just print current application.
Look at `oc project`, it works in the same way.

Reply:
@kadel Ah, that makes sense. Was a bit confused. I'll update the PR.

Reply:
no problem, thank ;-)